Library/Homebrew/manpages/brew.1.md.erb

```diff
@@ -105,6 +105,12 @@ can take several different forms:
   * `HOMEBREW_BOTTLE_DOMAIN`:
     If set, instructs Homebrew to use the given URL as a download mirror for bottles.
 
+  * `HOMEBREW_ARTIFACT_DOMAIN`:
+    If set, instructs Homebrew to use the given URL as a download mirror for bottles and binaries.
+    
+    *Note:* provide repository name in URL like
+    `HOMEBREW_ARTIFACT_DOMAIN=https://{URL of Artifact Manager}/{Repository name}`
```

Comment:
This implementation doesn't allow you to differentiate between bottled and non-bottled. It's worth just recommending a repository name here e.g. `homebrew`